src/ServiceFabric.Mocks/ServiceFabric.Mocks.csproj

```diff
@@ -21,7 +21,7 @@
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <IncludeSymbols>true</IncludeSymbols>
-    <Platforms>x64</Platforms>
+    <Platforms>AnyCPU</Platforms>
```

Comment:
Service Fabric is x64 only, so please revert.

Reply:
Reverted as requested.